streamx-console/streamx-console-service/src/main/java/com/streamxhub/streamx/console/core/entity/Application.java

```diff
@@ -793,6 +798,11 @@ public static Dependency toDependency(String dependency) {
             return new Dependency();
         }
 
+        public void extractJarMd5() {
```

Comment:
Compare whether the two files are the same. Take a look at the eaquls method in FileUtils. Using MD5 will first read all bytes into memory and calculate MD5. This method is not efficient, so it is not recommended